boot-core/src/lib.rs

```diff
@@ -45,39 +46,4 @@ pub(crate) mod cosmos_modules {
     pub use cosmrs::proto::cosmos::tx::v1beta1 as tx;
     pub use cosmrs::proto::cosmwasm::wasm::v1 as cosmwasm;
     pub use cosmrs::proto::tendermint::abci as tendermint_abci;
-}
-
-// mod macro_dev {
-//     use terra_rust_script_derive::contract;
-
-//     #[derive(Clone, Debug, contract)]
```

Comment:
Is this what the `terra_rust_script_derive::contract` had done before?

Reply:
Yes and no, the old macro just returned the ExecuteMsg variant. This generates a trait and implements it for any contract with that ExecuteMsg. The `payable` attribute lets you specify which functions should accept a payment. All the generated functions can then be called on any boot-object (with any boot-compatible backend environment)

Reply:
I've been waiting with writing our testing infrastructure for this haha

Reply:
> Yes and no, the old macro just returned the ExecuteMsg variant. This generates a trait and implements it for any contract with that ExecuteMsg. The `payable` attribute lets you specify which functions should accept a payment. All the generated functions can then be called on any boot-object (with any boot-compatible backend environment) 

Yeah the payable attribute seems really versatile and smart contract authors can use it as a reference to add others ~